tools/deb/istio-iptables.sh

```diff
@@ -0,0 +1,111 @@
+#!/bin/bash
```

Comment:
+1. We really grokked on this script back in Feb. I would really like to understand what is different here, and make sure that we don't end up having to deal with two iptable variants without understanding what the heck is going on. In fact, I suggest that if possible, we have a single script with if platform==k8s else.. (where platform can be set in the beginning) and keep that script in pilot.